docs/DeveloperGuide.md

```diff
@@ -168,6 +168,40 @@ Step 4. The `Ui` calls `UiCommand#execute(MainWindow)`, which will result in a c
     * Cons: 
         * `Ui` and `CommandResult` are not closed to modification. A new instruction might require the addition of fields to `CommandResult` (to store instructions and related data) as well as a new conditional statement in `Ui` to handle the new instruction.
 
+### Add Event to Project Command [Implemented in v1.2]
+
+The mechanism is used to add an event to the `EventList` of `Project` specified by the index in the project list shown.
+
+The client creates a concrete `AddEventCommand` that contains the specified index of project and a valid Event object. Each concrete `AddEventCommand` implements the `AddEventCommand#execute` method, which calls the appropriate method(s) in `Project` to update its event list and appropriate method(s) in `Model` to update the Project List.
+
+Given below is an example usage scenario and how the mechanism behaves at each step.
+
+Step 1. The user issues ececutes the command `addEto 1 d/Tutorial i/WEEKLY at/25-03-2021`, which adds an `Event` with description, interval and date specified to `Project` 1 in Project List.
+
+Step 2: The input is parsed by `AddEventCommandParser`. It checks `Event` provided is valid or not. If input is invalid, an exception will be thrown, `Ui` will help print out the exception message. Otherwise, an `AddEventCommand` will be created.
```

Comment:
It checks `Event` provided is valid or not -> It checks if `Event` provided is valid or not

Reply:
"an exception will be thrown, `Ui` will" change to "an exception will be thrown and `Ui` will"